Resources/Prototypes/_StarLight/Entities/Objects/Weapons/Guns/Projectiles/hitscan.yml

```
@@ -1028,4 +1028,18 @@
     state: pd_trace
   impactFlash:
     sprite: _Starlight/Objects/Weapons/Guns/Projectiles/projectiles.rsi
-    state: pd_impact
\ No newline at end of file
+    state: pd_impact
+
+# Xeno
+
+- type: hitscan
+  id: BulletAcidHitscan
+  name: acid spit
+  bullet:
+    sprite:
+      sprite: Objects/Weapons/Guns/Projectiles/xeno_toxic.rsi
+      state: xeno_toxic
+  collisionMask: 64 #BulletImpassable
+  damage:
+    types:
+      Caustic: 5
```

```suggestion
      Caustic: 8
```
Having to hit 20 times to kill with that rate of fire is definitely quite a challenge.